### ISSUE TITLE
v0.2.5 run-dialog never appears on screen

### ISSUE THREAD

**Issue by scorpp** [hbekel/bashrun2#6]
```
$ rm -rf ~/.cache/bashrun2/
$ bashrun2 -d show
install: creating directory '/home/user/.cache/bashrun2'
create_modeinfo using /home/user/.config/bashrun2/bashrun2.rc 
+plugins terminal -> /home/user/.config/bashrun2/plugins/terminal.rc 
config.restore +configure-terminal-modes -> reconfigure... 
config.restore restore modes-core.dump 
config.restore calling +configure-terminal-modes 
config.restore dumping modes-core-terminal.dump 
config.restore caching modes-core-terminal.rc 
config.restore +configure-modes -> reconfigure... 
config.restore calling +configure-modes 
config.restore dumping modes-core-terminal-user.dump 
config.restore caching modes-core-terminal-user.rc 
config.restore restore templates-core.dump 
terminal.expand expanded -> xterm -title 'bashrun2' -name bashrun2-run-dialog -g 80x24 -fg grey -bg black +sb -xrm XTerm**metaSendsEscape:True -xrm XTerm**allowWindowOps:True -e bash --rcfile /usr/share/bashrun2/bashrc 
bashrun=1             bashrun_frontend=1             bashrun_mode=run-dialog        bashrun_feedback=1             bashrun_debug=1             xterm -title 'bashrun2' -name bashrun2-run-dialog -g 80x24 -fg grey -bg black sb -xrm XTerm**metaSendsEscape:True -xrm XTerm**allowWindowOps:True -e bash --rcfile /usr/share/bashrun2/bashrc
launch The terminal window never appeared... giving up. 
```

bashrun2 is v0.2.5
bash is 5.1.0(1)-release
xterm is 362
ArchLinux

**Comment by hbekel** [comment]
I can reproduce this when I switch my bashrun2 to use xterm.

I notice that xterm 362 does not even start when trying to run it on my updated arch install (same specs as yours).

pacman -S xterm

-> shows that some of its files "exist in filesystem"

pacman -S --override '*'

-> installs xterm, and xterm works again (on it's own and with bashrun).

So this seems to be an issue with arch's xterm (?)

**Comment by scorpp** [comment]
I tried to reinstall xterm, but in my case pacman showed no conflicts. I still removed it with all deps (`-Rcs`) and installed back, but no luck.

i tried to run xterm cmd line as printed by debug output (`terminal.expand expanded ->`) directly and the window popped up
 ```
xterm -title 'bashrun2' -name bashrun2-run-dialog -g 80x24 -fg grey -bg black +sb -xrm XTerm**metaSendsEscape:True -xrm XTerm**allowWindowOps:True -e bash --rcfile /usr/share/bashrun2/bashrc
```

is there anything else to try?

**Comment by hbekel** [comment]
I have meanwhile found another culprit, please try with the latest commit. The line below the teminal.expand debug output is the one actually executed, and there the +sb option has it's plus sign removed.

**Comment by scorpp** [comment]
yes! it is it! works back again!

**Comment by hbekel** [comment]
Guess I'll have to make another release, then. Thank you for your help!

**Comment by scorpp** [comment]
thank you for the fix!

**Comment by hbekel** [comment]
Fixed in Version 0.2.6
